lib/browser/guest-view-manager.js

```diff
@@ -184,7 +184,6 @@ const attachGuest = function (event, elementInstanceId, guestInstanceId, params)
     guestInstanceId: guestInstanceId,
     nodeIntegration: params.nodeintegration != null ? params.nodeintegration : false,
     plugins: params.plugins,
-    zoomFactor: embedder.getZoomFactor(),
```

Comment:
What do you think about using `embedder._getZoomFactor()` here so that below we don't have to wrap it in side a callback?